### PR TITLE
Make the Vagrant Setup Instructions easier to find

### PR DESCRIPTION
**Pull request by mikedep333**
I am incorporating the feedback from @moollaza in PR #213

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ We run our DuckPAN tests against Perl 5.16 and 5.18 using Travis (https://travis
     + [Installing the Virtual Machine](#installing-the-virtual-machine)
     + [Using the Virtual Machine](#using-the-virtual-machine)
 - [Vagrant Virtual Environment](#vagrant-virtual-environment)
+    + [Setup Instructions](#setup-instructions-2)
```
Comment (moollaza) [inline README.md]: whoops, looks like there's an errant `-2` in the link?

**Comment by moollaza** [comment]
@mikedep333 thanks a lot! One small typo in the link URL, otherwise perfect

File: README.md (hunk 1 above)
Comment (mikedep333) [inline README.md]: I was wondering about that myself. The old version had it. 3e09d6966208b73cb89cdbc77f93ad8c0f08057f
I will fix it though.

**Comment by moollaza** [comment]
LGTM :+1:

**Pull request merged**